src/pools/D3MAaveDai.t.sol

```diff
@@ -782,8 +731,8 @@ contract DssDirectDepositAaveDaiTest is DSTest {
 
         end.thaw();
 
-        // Unwind via exec should fail with error "DssDirectDepositAaveDai/end-debt-already-set"
-        deposit.exec();
+        // Unwind via exec should fail with error "D3MAaveDai/end-debt-already-set"
+        directDepositHub.exec(ilk);
     }
 
     function test_unwind_culled_then_mcd_caged() public {
```

Comment:
should add tests that:

1. cages d3m
2. partial unwind
3. dss gets caged
4. d3m gets liquidity
5. unwind continues

cull/uncull

1. cages d3m
2. partial unwind
3. cull
4. dss gets caged
5. uncull
6. d3m gets liquidity
7. unwind continues

Reply:
will handle these here: https://github.com/makerdao/dss-direct-deposit/pull/70